tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -400,6 +401,7 @@
       .datapoint-right-controls-holder {
         display: flex;
         flex-direction: row-reverse;
+        overflow: hidden;
```

Comment:
I think it is more correct to put `flex-shrink: 0;`.

Reply:
I did try that but it doesn't work. With this container having flex-shrink: 0, the child div with lots of text will still cause this container to grow in width beyond the screen, leading to the issue.

Reply:
moved to flex-shrink: 0 for the two buttons as you suggested and removed the min-width setting for the button.

still need the overflow: hidden on the right-controls-holder to avoid overflow.

Reply:
Yup, understand that now.